cfgov/jinja2/v1/_includes/molecules/featured-menu-content.html

```diff
@@ -31,8 +31,7 @@
     {% set img_src = featured_image.url if featured_image.url else '' %}
 {% else %}
     {% set featured_image = value.image %}
-    {% set img_src = static(featured_image.url) if featured_image.url else '' %}
-
+    {% set img_src = featured_image.url if featured_image.url else '' %}
```

Comment:
This logic was unused, and was a leftover of the days before the mega menu was in Wagtail. I made this change to support the existing menu design with the new backend; when the menu content contains an image URL, we shouldn't assume that it'll be in Django static files. It never is because featured menu images are Wagtail image uploads, not static files.